### DISCUSSION TITLE
Add link to existing text

### DISCUSSION THREAD

**Discussion by Opfour** [tsujan/FeatherPad#545] [Q&A]
Hi, 

Not so much an issue, but more of a question... 
I see I can copy/pasta text that contains links which shows up correctly, but is it possible to add a link to typed text within FP?

**Comment by tsujan** [comment]
Any valid URL (http:///..., file:///..., etc.) is automatically a link that can be opened from inside FeatherPad. Hidden links — having a text different from the link address — aren't supported because they aren't in plain-text.
